src/charts/stackedArea/StackedAreaComponent.js

```diff
@@ -58,6 +59,8 @@ export default class StackedArea extends React.Component {
     componentDidUpdate() {
         this.props.chart.destroy(this._rootNode);
         this.props.chart.create(this._rootNode, this.props.data, this._getChartConfiguration());
+
+        this.props.createTooltip();
```

Comment:
@miglesiasEB This is the only solution I've been able to find to making the tooltip work correctly with the `withResponsiveness` HOC we made.